R/is_something.R

```diff
@@ -19,30 +19,30 @@ is_parenthesized <- function(expr)
   is.call(expr) && identical(expr[[1L]], quote(`(`))
 }
 
-# Check whether a pipe is a tee.
+# Check whether a pipe is the tee pipe operator.
 #
 # @param pipe A (quoted) pipe
 # @return logical - TRUE if pipe is a tee, FALSE otherwise.
-is_tee <- function(pipe)
+is_tee_pipe <- function(pipe)
```

Comment:
Can you please revert the changes to function names? That would be a breaking change, and I don't think it's worth the cost.

Reply:
OK

Note: are they used in other repositories? I made sure to change them in the entire repository. So I don't think that I broke anything.

Reply:
Any changes to functions require that we fully understand the implications to other packages.

Reply:
OK. Will revert then.

Reply:
Done.